### PR TITLE
Fix a number of pylint warnings

### PR DESCRIPTION
**Pull request by brianphaley**
Fixed code to address a number of pylint warnings. Also removed some items from the disabled list as
they were not triggering any warnings.

### CODE REVIEW

File: hotsos/core/plugins/storage/ceph.py [hunk 1]
```diff
@@ -965,15 +965,15 @@ def days_to_eol(self):
             delta = (eol - today).days
             return delta
 
-    def _get_bind_interfaces(self, type):
+    def _get_bind_interfaces(self, intf_type):
```
Comment (dosaboy) [inline hotsos/core/plugins/storage/ceph.py:968]: i think the convention used elsewhere would be "iface_type"

File: hotsos/cli.py [hunk 2]
```diff
@@ -120,11 +120,11 @@ def get_templates_path():
 
 
 def spinner(msg, done):
-    spinner = Spinner(msg)
+    my_spinner = Spinner(msg)
```
Comment (dosaboy) [inline hotsos/cli.py:123]: maybe cli_spinner?

File: hotsos/core/plugintools.py [hunk 3]
```diff
@@ -132,9 +132,7 @@ def _expand_dict(self, data, level):
         return markdown_output
 
     def _expand_list(self, data):
-        markdown_output = '\n'
-        for item in data:
-            markdown_output += f'- {item}\n'
+        markdown_output = '\n%s' % ''.join(f'- {item}\n' for item in data)
```
[the remark this answers is not in the document]
Comment (brianphaley) [reply hotsos/core/plugintools.py:135]: It didn't seem to in my testing with ['foo', 'bar', 'baz']

--> python ./expand_list.py 
old:" 
- foo
- bar
- baz
 "
new:" 
- foo
- bar
- baz
 "

[the remark this answers is not in the document]
Comment (dosaboy) [reply hotsos/core/plugintools.py:136]: sorry you are right, my test was wrong.

**Pull request merged**